packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -231,10 +248,17 @@ class _CupertinoCheckboxState extends State<CupertinoCheckbox> with TickerProvid
     final Color effectiveCheckColor = widget.checkColor
       ?? CupertinoColors.white;
 
+    final WidgetStateProperty<MouseCursor> effectiveMouseCursor =
```

Comment:
After updating the parameter we can assign `widget.mouseCursor` and followed by default.

Like
`final WidgetStateProperty<MouseCursor> effectiveMouseCursor = widget.mouseCursor ??  <Default>;`

Reply:
If I understand correctly does this mean it should be rewritten as
```
final MouseCursor effectiveMouseCursor = widget.mouseCursor ??      
    WidgetStateProperty.resolveWith<MouseCursor>((Set<WidgetState> states) {
        return states.contains(WidgetState.disabled)
              ? SystemMouseCursors.basic
              : kIsWeb ? SystemMouseCursors.click : SystemMouseCursors.basic
            );
```

Reply:
But `buildToggleable` expects `effectiveMouseCursor to be of type `WidgetStateProperty` not of type `MouseCursor`.

Reply:
I see. Thank you for explaining.